Cargo.toml

```diff
@@ -1,13 +1,10 @@
 [package]
-authors = [
-    "Carl Lerche <me@carllerche.com>",
-    "Andrew Cann <shum@canndrew.org>",
-]
+authors = ["MaidSafe Developers <dev@maidsafe.net>"]
 description = "tokio-based uTP implementation"
 documentation = "https://docs.rs/tokio-utp"
-license = "MIT/Apache-2.0"
-name = "tokio-utp"
-repository = "https://github.com/canndrew/tokio-utp"
+license = "MIT/BSD"
+name = "tokio_utp"
```

Comment:
also let's call this `tokio-utp` in line with all the `tokio` prefixed crates in `crates.io`. Here's a list from @povilasb for reference: https://docs.rs/releases/search?query=tokio

Our repo is still `tokio_utp` which is fine and in line with our convention

Reply:
updated 👍